avro/src/bigdecimal.rs

```diff
@@ -182,6 +182,46 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_avro_3779_record_with_bg_ser() -> TestResult {
```

Comment:
This issue number is not relevant for this test, please use `avro_rs_338` so it points to this PR